src/utils/jump_utils.jl

```diff
@@ -220,12 +220,20 @@ function _calc_dimensions(
     # 2. Read data for one execution for all devices.
     # This will ensure that data on disk is contiguous in both cases.
     if length(ax) == 1
-        dims = (horizon, 1, num_rows)
+        if length(ax[1]) != horizon
+            @debug "$(encode_key_as_string(key)) has length $(length(ax[1])). Different than horizon $horizon."
```

Comment:
Are debug messages appropriate here?

Reply:
I can change it to an Info, but I assumed the user would understand the behavior.